woocommerce/woocommerce-ios
# [Mobile Payments] Remember at most one card reader

Closes #4953 
Closes #4933 

Changes:
- Remembers at most one card reader in app settings
- "Forget" action no longer needs a card reader ID to forget (it just resets the array)

Note:
- For backwards compatibility, the storage in app settings is still an array
- If multiple reader IDs are found in app settings (i.e. persisted by a version prior to this change), just the last one added is returned

To test:
- Get two readers
- Initially reader A powered on, reader B off
- Connect to reader A (this will cause the app to "remember" just reader A)
- Exit the app
- Power off reader A, power on reader B
- Start the app and enter the connection flow. Search should begin automatically since there is a remembered reader
- Verify the app finds reader B and asks you if you'd like to connect to it. Connect to it.
- Exit the app
- Power on reader A, power off reader B
- Start the app and enter the connection flow. Search should begin automatically since there is a remembered reader
- Verify the app DOES NOT connect to reader A automatically any more (it should prompt you since it is no longer remembered - at this point only reader B is remembered)
- Do NOT connect to reader A
- Exit the app
- Power off reader A, power on reader B
- Start the app and enter the connection flow. Search should begin automatically since there is a remembered reader and it should connect automatically to reader B
- Disconnect from reader B
- Ensure search does not auto restart (since there are now no remembered readers)
- Exit the app
- Start the app and enter the connection flow. Search should NOT begin automatically since there is NO remembered reader

Update release notes:
